src/librustc/middle/stability.rs

```diff
@@ -515,7 +515,7 @@ pub enum EvalResult {
     Deny {
         feature: Symbol,
         reason: Option<Symbol>,
-        issue: u32,
+        issue: Option<u32>,
```

Comment:
What about using a `Option<NonZeroU32>>` instead? There's no valid issue #0, which means that we can try to save some space here.

Reply:
It can't be done in this PR though, because the stage 0 compiler expects the old syntax and therefore the `issue = "0"` in stdlib and stdcore must remain until the changes made here get into stage 0.
(To be precise, `x.py check` emits errors and AFAIK Travis checks that.)
Also, there are other places where issues are represented as `Option<u32>`